spec/policies/event_procedure_policy_spec.rb

```diff
@@ -3,75 +3,39 @@
 require "rails_helper"
 
 RSpec.describe EventProcedurePolicy do
-  describe "Scope" do
-    it "returns all event_procedures for user" do
-      user = create(:user)
-      event_procedure = create(:event_procedure, user: user)
+  let(:user) { create(:user) }
+  let(:event_procedure) { create(:event_procedure, user: user) }
+  let(:event_procedure_without_user) { create(:event_procedure) }
```

Comment:
this event procedure has user let's rename it to `other_user_event_preocedure`

Reply:
Done! f9c127650d08c779d0ac2c8c204fc2c96036c006